tycho-extras/tycho-eclipserun-plugin/src/main/java/org/eclipse/tycho/extras/eclipserun/EclipseRunMojo.java

```diff
@@ -151,8 +150,8 @@ public class EclipseRunMojo extends AbstractMojo {
 	 * 
 	 * <pre>
 	 * &lt;jvmArgs&gt;
-	 *   &lt;args&gt;-Xdebug&lt;/args&gt;
-	 *   &lt;args&gt;-Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=1044&lt;/args&gt;
+	 *   &lt;arg&gt;-Xdebug&lt;/arg&gt;
```

Comment:
When you edit such javadoc would you please make it :
```
{@code
<jvmArgs>
	 <args>-Xdebug</args>
	   <args>-Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=1044</args>
 </jvmArgs>
 }
```
That makes it far more readable.

Reply:
Fine with me.

Reply:
You cna use `@Parameter(alias="applicationsArgs")` to keep compatibility while renaming the field.